R/derive_var_ontrtfl.R

```diff
@@ -271,7 +281,7 @@ derive_var_ontrtfl <- function(dataset,
   }
 
   # scenario 4: end_date and span_period are passed
-  if (!is.null(span_period)) {
+  if (isTRUE(span_period)) {
```

Comment:
```suggestion
  if (span_period) {
```
Is the `isTRUE` redundant?

Reply:
I find it easier to read, so i don't need to remember that span_period is lgl. But you're 100% correct it is not needed